zerver/webhooks/zapier/view.py

```diff
@@ -6,14 +6,31 @@
 from zerver.decorator import api_key_only_webhook_view
 from zerver.lib.request import REQ, has_request_variables
 from zerver.lib.response import json_error, json_success
-from zerver.lib.webhooks.common import check_send_webhook_message
+from zerver.lib.webhooks.common import check_send_webhook_message, \
+    validate_extract_webhook_http_header
 from zerver.models import UserProfile
 
 @api_key_only_webhook_view('Zapier', notify_bot_owner_on_invalid_json=False)
 @has_request_variables
 def api_zapier_webhook(request: HttpRequest, user_profile: UserProfile,
                        payload: Dict[str, Any]=REQ(argument_type='body')) -> HttpResponse:
 
+    # A request with the ZapierZulipApp user agent is a request from
+    # the official Zulip app for Zapier
```

Comment:
We should be clear here that "this is a sanity check for whether this is a request from the official Zulip app for Zapier".  

User-agents are easily spoofed, so we definitely don't _know_ that this is that app, and our comments should avoid suggesting we do.